dbus/dbus.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2020 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+package dbus
+
+import "github.com/pkg/errors"
+
+var dbusInterface DBusInterface = nil
```

Comment:
Isn't it better to return this from a function instead and allow callers to decide whether it should be global or not?

Reply:
Sorry I don't understand your question. It is returned from `GetDBusInterface()`, it is only on the same package that is accessible.

Reply:
Alright, I got it, just rename to DBusAPI to avoid confusion around the concept which is "DBus interface".

Reply:
:+1: